### PR TITLE
Git and Bash

### PR DESCRIPTION
**Pull request by prachi610**
<hr>

## Description 📜
I have added the documentation of Git and Bash. In which I have explain about Git and Bash and added some commands .

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below)

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by Pooj123-A** [comment]
**/assign**

**Comment by praveenscience** [comment]
@Pooj123-A Since you asked for it, can you please send me a report by tomorrow afternoon as now it's assigned to you. Please let me know your review comments.

**Comment by Pooj123-A** [comment]
Praveen I don't understand that what do you mean by report.
Do I need to complete it by tomorrow ?

**Review by ipsitagoel — COMMENTED** [review]
@prachi610 Good job!
@2024-SANDHYA You can have a look and merge this!

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @prachi610 Good job! @2024-SANDHYA You can have a look and merge this!

Sure!

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Nice! Just one change @prachi610, remove your name from the bottom & remove this too, please. The rest looks great!

image

**Comment by prachi610** [comment]
@2024-SANDHYA Done!

File: Web_Development/Javascript/Git and Bash/readme.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+- WSL is very handy and flexible for development-related purposes in a Linux-like environment.
+### Final verdict
+
+- Both git and bash give you a powerful and flexible environment for any kind of workflow and development.
+- Windows version of git comes with a bash shell, but you can only perform the basic tasks with that version of the shell.
+- I would prefer you all to use WSL with git which gives you a separate Linux environment for your development where you can use all of the features of bash as well as git too.
+
+
+
+### Contributed By [Prachi Singh]("https://github.com/prachi610/)
```
Comment (kashishahuja2002) [inline Web_Development/Javascript/Git and Bash/readme.md]: Remove this line.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

**Comment by kashishahuja2002** [comment]
Great work @prachi610

File: Web_Development/Javascript/Git and Bash/readme.md (hunk 1 above)
Comment (prachi610) [reply Web_Development/Javascript/Git and Bash/readme.md]: Check it now.

**Comment by prachi610** [comment]
> Great work @prachi610

Thanks!

**Review by 2024-SANDHYA — APPROVED** [review]

**Pull request merged**